Text Utilities/textUtilities.css

```diff
@@ -0,0 +1,58 @@
+@import url("../../Utility CSS/navbar.css");
+
+.code-container {
+  width: 100%;
+}
+
+.box {
+  display: flex;
+  flex-direction: column;
+}
+
+.h1,
+.h2,
+.h3,
+.h4,
+.h5,
+.h6 {
+  font-weight: bold;
+}
+
+.h1 {
+  font-size: 6.5rem;
+}
+
+.h2,
+.paragraph-xl {
+  font-size: 5.5rem;
+}
+
+.h3,
+.paragraph-lg {
+  font-size: 4.5rem;
+}
+
+.h4,
+.paragraph-sm {
+  font-size: 3.5rem;
+}
+
+.h5 {
+  font-size: 2.9rem;
+}
+
+.h6,
+.paragraph-xsm {
+  font-size: 2.3rem;
+}
+
+.grey-text {
+  font-size: 2.5rem;
+  font-weight: bold;
+  color: var(--btn-secondary-color);
+}
+
+.centered-text {
+  font-size: 4.5rem;
```

Comment:
Don't add `font-size` when it comes to centering text.

Reply:
OK I will keep it in mind. Thanks for the suggestion.